scripts/tableGenerator.js

```diff
@@ -1578,25 +1578,20 @@ function generateCart() {
         message += `${name} : ${item.quantity}Kg \n`;
     });
     if (currentOrders.length === 0) {
-        window.alert('There are no orders placed');
+        window.confirm('There are no orders placed');
         return '';
     } else {
-        window.alert(`${message}`);
+        result = window.confirm(`${message}`);
+        if (result === false) {
+            return;
+        }
+        const link = `https://api.whatsapp.com/send?phone=${phone}&text=${message}`;
+        window.open(link, '_blank');
     }
 
     return message;
 }
 
-function sendOrder() {
-    const message = encodeURI(generateCart());
```

Comment:
This can be something like this `const message = getOrderMessage();` 
`if (!message) //currently there are no orders`
`else // code for showing alert/confirm or a modal as well`

Reply:
Why is this necessary when the code can be compressed?